packages/playground/src/views/minting_view.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+  <view-layout>
+    <h2 class="mb-5 text-capitalize">Alpha software - all data subject to change</h2>
+    <v-form @submit.prevent="mintingHash()" class="d-inline-flex w-100">
+      <FormValidator v-model="isValidForm">
+        <InputValidator
+          :value="receiptHash"
+          :rules="[
+            validators.required('Receipt hash is required.'),
+            validators.equal('Receipt hash must be 64 characters long.', 64),
+            validators.isHash('Invalid hash', 'sha256'),
+          ]"
+          valid-message="Receipt hash is valid."
+          #="{ props: validationProps }"
+          ref="hashInput"
+        >
+          <VTextField
+            class="mr-5"
+            placeholder="Please insert your receipt hash"
+            v-model="receiptHash"
+            v-bind="{ ...validationProps }"
+            :loading="loading"
+        /></InputValidator>
+
+        <VBtn
+          type="submit"
+          color="primary"
+          variant="elevated"
+          :disabled="!isValidForm"
+          :loading="loading"
+          size="x-large"
+          rounded="lg"
+        >
```

Comment:
```suggestion
        <VBtn
          type="submit"
          color="primary"
          variant="tonal"
          :disabled="!isValidForm"
          :loading="loading"
          size="x-large"
          rounded="lg"
        >
```

Just updating the variant to `tonal` as it's used in the entire project
If there is a design which using elevated please skip this update